modules/central-services-public-config.adoc

```diff
@@ -271,52 +271,166 @@ When you disable autoscaling, the `minReplicas` and `maxReplicas` parameters do
+
+|===
+
+[id="central-services-public-configuration-file-scannerv4_{context}"]
+== Scanner V4
+The following table lists the configurable parameters for Scanner V4.
+
+|===
+| Parameter | Description
+
+| `scannerV4.db.persistence.persistentVolumeClaim.claimName`
+|The name of the PVC to manage persistent data for Scanner V4.
+If no PVC with the given name exists, it will be created. The default value is `scanner-v4-db` if not set. To prevent data loss, the PVC is not removed automatically with Central`s deletion.
+
+| `scannerV4.disable`
+| Use `false` to enable Scanner V4. When setting this parameter, the StackRox Scanner must also be enabled by setting `scanner.disable=false`. Until feature parity between the StackRox Scanner and Scanner V4 is reached, Scanner V4 can only be used in combination with the StackRox Scanner. Enabling Scanner V4 without also enabling the StackRox Scanner is not supported. When you set this parameter to `true` with the `helm upgrade` command, Helm removes the existing Scanner V4 deployment.
+
+| `scannerV4.exposeMonitoring`
+| Specify `true` to expose Prometheus metrics endpoint for Scanner V4 on port number `9090`.
+
+| `scannerV4.indexer.replicas`
+| The number of replicas to create for the Scanner V4 Indexer deployment.
+When you use it with the `scannerV4.indexer.autoscaling` parameter, this value sets the initial number of replicas.
+
+| `scannerV4.indexer.logLevel`
+| Configure the log level for the Scanner V4 Indexer.
+Red Hat recommends that you not change the log level's default value (`INFO`).
+
+| `scannerV4.indexer.nodeSelector`
+| Specify a node selector label as `label-key: label-value` to force the Scanner V4 Indexer to only schedule on nodes with the specified label.
+
+| `scannerV4.indexer.tolerations`
+| If the node selector selects tainted nodes, use this parameter to specify a taint toleration key, value, and effect for the Scanner V4 Indexer. This parameter is mainly used for infrastructure nodes.
+
+| `scannerV4.indexer.autoscaling.disable`
+| Use `true` to disable autoscaling for the Scanner V4 Indexer deployment.
+When you disable autoscaling, the `minReplicas` and `maxReplicas` parameters do not have any effect.
+
+| `scannerV4.indexer.autoscaling.minReplicas`
+| The minimum number of replicas for autoscaling.
+
+| `scannerV4.indexer.autoscaling.maxReplicas`
+| The maximum number of replicas for autoscaling.
+
+| `scannerV4.indexer.resources.requests.memory`
+| The memory request for the Scanner V4 Indexer to override the default value.
+
+| `scannerV4.indexer.resources.requests.cpu`
+| The CPU request for the Scanner V4 Indexer to override the default value.
+
+| `scannerV4.indexer.resources.limits.memory`
+| The memory limit for the Scanner V4 Indexer to override the default value.
+
+| `scannerV4.indexer.resources.limits.cpu`
+| The CPU limit for the Scanner V4 Indexer to override the default value.
```

Comment:
For all of these, do we need "to override the default value"? That's generally what the user is doing when they change a parameter value.